lib/Utils/dir_list_ci.cpp

```diff
@@ -55,59 +103,144 @@ std::string DirListCI::resolveFileCase(const std::string &in_name)
     auto subDir = name.find('/');
     if(subDir != std::string::npos)
     {
+        // THIS BEHAVIOR IS EXPENSIVE AND WASTEFUL, AVOID INVOKING AT ALL COSTS!!!
         auto sdName = resolveDirCase(name.substr(0, subDir));
         DirListCI sd(m_curDir + "/" + sdName);
-        return sdName + "/" + sd.resolveFileCase(name.substr(subDir + 1));
+        std::string found = sd.resolveFileCase(name.substr(subDir + 1));
+        if(found.empty())
+            return "";
+        else
+            return sdName + "/" + found;
```

Comment:
This thing is mainly used by music paths as they usually stored at any random folder, and going be relative to the episode directory always.

Reply:
Thank you for the info. I knew that most cases would not invoke the subdir search, but I didn't know when it would be used.

Reply:
There are many episodes that links music at subdirectories, the significant examples are A2XT games that directly require contributors to put music into sub-directory